modules/react-map-gl-draw/src/types.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+export type {
```

Comment:
why you need this? Can you just import the types from `@nebula.gl/edit-modes` directly?